Makefile

```diff
@@ -132,10 +132,10 @@ build:
 	@CGO_ENABLED=0 GOOS=linux GO111MODULE=on go build -a -o manager main.go
 
 build-test-image:
-	docker build -t quay.io/$(QUAY_USERNAME)/my-mongodb-operator:test \
+	docker build -t icr.io/$(ICR_USERNAME)/my-mongodb-operator:test \
 	--build-arg VCS_REF=$(VCS_REF) --build-arg VCS_URL=$(VCS_URL) \
 	--build-arg GOARCH="amd64" -f Dockerfile .
-	docker push quay.io/$(QUAY_USERNAME)/my-mongodb-operator:test
+	docker push icr.io/$(ICR_USERNAME)/my-mongodb-operator:test
```

Comment:
This build/push should probably remain on quay as this is just building/pushing a test image to the user's own quay repo for testing purposes.  End users aren't going to have access to build/push directly to anywhere in ICR.